src/main/java/com/google/api/codegen/discovery/config/ApiaryConfigToSampleConfigConverter.java

```diff
@@ -88,13 +88,21 @@ private MethodInfo createMethod(Method method) {
     // The order of fields must be preserved, so we use an ImmutableMap.
     ImmutableMap.Builder<String, FieldInfo> fields = new ImmutableMap.Builder<>();
     TypeInfo requestBodyType = null;
+    boolean isPageStreamingResourceSetterInRequestBody = false;
     for (String fieldName : apiaryConfig.getMethodParams(method.getName())) {
       Type containerType = apiaryConfig.getType(method.getRequestTypeUrl());
       Field field = apiaryConfig.getField(containerType, fieldName);
       // If one of the method arguments has the field name "request$", it's the
       // request body.
       if (fieldName.equals(DiscoveryImporter.REQUEST_FIELD_NAME)) {
         requestBodyType = createTypeInfo(field, method);
+
+        Type type = apiaryConfig.getType(field.getTypeUrl());
+        for (Field field2 : type.getFieldsList()) {
+          if (field2.getName().equals(PAGE_TOKEN_FIELD_NAME)) {
+            isPageStreamingResourceSetterInRequestBody = true;
+          }
+        }
```

Comment:
Sure, feel free to merge #880 whenever, I don't think there's anything else coming besides this